repos/base-linux/src/core/spec/linux/dataspace_component.cc

```diff
@@ -0,0 +1,77 @@
+/*
+ * \brief  Linux-specific core implementation of the dataspace component
+ * \author Stefan Kalkowski
```

Comment:
Is this the real author?

Reply:
Yes I think so. The file has originally been copied from `repos/base-linux/src/core/dataspace_component.cc`. Since `repos/base-linux/src/core/spec/pc/dataspace_component.cc` is also (an adapted) copy of this file git decided that this is the new file.